lib/util/http/types.ts

```diff
@@ -90,4 +90,5 @@ export interface HttpResponse<T = string> {
   body: T;
   headers: HttpHeaders;
   authorization?: boolean;
+  url: string;
```

Comment:
I think this isn't required anymore, because the source url you try to use is wrong anyways